src/styles/app.scss

```diff
@@ -324,3 +377,37 @@ header h1 {
   }
 }
 
+// Theme switch popup
+// theme selector visible only if JavaScript is available
+
+.theme-container {
+  position: relative;
+}
+
+.theme-icon {
+  display: none;
+  text-align: center;
+  width: 2em;
+  height: 2em;
+  line-height: 2em;
+  border: 1px solid var(--theme-icon);
+  border-radius: 5px;
+  user-select: none;
+  cursor: pointer;
+}
+
+.theme-choice {
+  display: none;
+  border: 2px solid var(--theme-popup-border);
+  border-radius: 5px;
+  color: var(--theme-choice-color);
+  background: var(--theme-popup-bg);
+  position: absolute;
+  list-style: none;
+  margin: 10px 10px 0px 0px;
```

Comment:
Please remove this. The menu doesn't need margin since it's `position: absolute`.

Reply:
these margins cannot be removed. There is a lot of overlapping styling CSS (I start to think _too much_ styling) and this is the only way to have the theme selection dropdown stay in place.